rules/windows/execution_from_unusual_path_cmdline.toml

```
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/10/30"
+maturity = "production"
+updated_date = "2020/10/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution from suspicious default Windows directories. This is sometimes done by adversaries to hide
+malware in trusted paths.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution from Unusual Directory - CommandLine"
```

```suggestion
name = "Execution from Unusual Directory - Command Line"
```
we use a space in most of our rules for Command Line